packages/providers/src/provider.ts

```diff
@@ -190,6 +190,10 @@ export type BuildPredicateOptions = {
   fundTransaction?: boolean;
 } & Pick<TransactionRequestLike, 'gasLimit' | 'gasPrice' | 'maturity'>;
 
+export type BuildScriptOptions = {
+  fundTransaction?: boolean;
+} & Pick<TransactionRequestLike, 'gasLimit' | 'gasPrice' | 'maturity'>;
+
```

Comment:
@camsjams It looks like `BuildScriptOptions` is not used anywhere. 

Or is it?